web/themes/custom/pineapple/sass/layouts/nodes/_article-teaser-page.scss

```diff
@@ -0,0 +1,51 @@
+.views-row {
+  margin-bottom: 16px;
+}
+
+.teaser-container {
```

Comment:
Why create a new container we you can target existing classes on the `<article>` tag

Reply:
absolutely. I created it for a reason when I had a different markup and forgot to remove.